src/plugins/timeConductor/ConductorMode.vue

```diff
@@ -180,12 +180,13 @@ export default {
         },
 
         toggleMenu(event) {
+            event.preventDefault();
             this.showMenu = !this.showMenu;
 
             if (this.showMenu) {
-                document.addEventListener('click', this.toggleMenu, true);
+                setTimeout(() => document.addEventListener('click', this.toggleMenu));
```

Comment:
You're right, I'll take a look at it.

Reply:
There is a toggle mixin in the toolbar which solves this same problem for the menus there.  At this point it could be moved to a place for general platform reuse.